src/test/java/io/openliberty/tools/intellij/it/GradleSingleModMPProjectTest.java

```diff
@@ -88,6 +93,16 @@ public String getWLPInstallPath() {
         return WLP_INSTALL_PATH;
     }
 
+    /**
+     * Returns the path of server.env file.
+     *
+     * @return The path of server.env file.
+     */
+    @Override
+    public String[] getWLPServerPath() {
```

Comment:
Give more meaningful name.

Reply:
Changed the method name to getServerEnvPath()